.github/workflows/trunk-cd.yml

```diff
@@ -43,7 +62,7 @@ jobs:
           name: published
           path: result.json
 
-  smoke-test:
+  e2e-tests:
```

Comment:
e2e tests are still relevant. They give us more confidence in our _stable_ releases.